.github/workflows/gradle-plugin.yml

```diff
@@ -22,7 +23,7 @@ jobs:
         run: |
           cd gradle-plugins
           ./gradlew assemble
-          ./gradlew :compose:check --continue
+          ./gradlew --continue :preview-rpc:test :compose:test :compose:testGradle-${{ matrix.gradle }}
```

Comment:
should we increase number of gradle workers?
`--max-workers     Configure the number of concurrent workers Gradle is allowed to use.`

Or it has no sence on GitHub Actions for now?

Reply:
Does not make sense on GHA